cypress/e2e/EngageSphere.cy.js

```diff
@@ -0,0 +1,41 @@
+describe('Test suite for Engage Sphere Website', () => {
+
+  beforeEach(() => {
+    cy.visit(Cypress.config('baseUrl'))
```

Comment:
Ao definir a `baseUrl` no arquivo de configurações do Cypress (`cypress.config.js`), a mesma pode ser omitida ao executar o comando `cy.visit()`, conforme comentei na aula 2 ao apresentar a [documentação oficial de tal comando](https://docs.cypress.io/api/commands/visit#Prefixes).

```suggestion
    cy.visit('')
```